src/runmodes.c

```diff
@@ -157,6 +156,13 @@ static const char *RunModeTranslateModeToName(int runmode)
 #else
             return "WINDIVERT(DISABLED)";
 #endif
+#ifdef HAVE_DPDK
```

Comment:
`ifdef` should come after `case` as for WINDIVERT for instance

Reply:
Fixed, thanks!